### PR TITLE
482: Empty state in consolidated cases

### PR DESCRIPTION
**Pull request by ericsorenson**

### CODE REVIEW

File: web-client/src/presenter/computeds/caseDetailHelper.js [hunk 1]
```diff
@@ -121,10 +121,14 @@ export const caseDetailHelper = (get, applicationContext) => {
     orderDesignatingPlaceOfTrial,
   ].some(hasOrder => !!hasOrder);
 
+  const hasNoConsolidatedCases =
+    !caseDetail.consolidatedCases || caseDetail.consolidatedCases.length === 0;
```
Comment (kkoskelin) [inline web-client/src/presenter/computeds/caseDetailHelper.js]: Could consider lodash's `isEmpty(caseDetail.consolidatedCases)`

File: web-client/src/presenter/computeds/caseDetailHelper.js [hunk 2]
```diff
@@ -121,10 +121,14 @@ export const caseDetailHelper = (get, applicationContext) => {
...
+
   return {
     caseCaptionPostfix: Case.CASE_CAPTION_POSTFIX,
     caseDeadlines,
     documentDetailTab,
+    hasNoConsolidatedCases,
```
Comment (kkoskelin) [inline web-client/src/presenter/computeds/caseDetailHelper.js]: philosophical question: how do we feel about computeds and properties whose name suggests a negation? Can get confusing if someone does `!hasNoConsolidatedCases && <el/>`

**Review by kkoskelin — APPROVED** [review]
I'm just your friendly local style police. Nothing's actually wrong here. :)

File: web-client/src/presenter/computeds/caseDetailHelper.js (hunk 2 above)
Comment (matthopson) [reply web-client/src/presenter/computeds/caseDetailHelper.js]: I agree.

**Pull request merged**